tests/e2e/testcases/subnet/public/suite.go

```diff
@@ -79,5 +79,13 @@ var _ = ginkgo.Describe("[Public Subnet]", func() {
 		gomega.Expect(err).Should(gomega.BeNil())
 		err = utils.RunHardhatTests(utils.BaseTest)
 		gomega.Expect(err).Should(gomega.BeNil())
+
+		// this is a simulation, so app is probably saving the info in the local sidecar section...TODO IS THIS SO?
```

Comment:
not sure what to do with this comment

Reply:
We are using the local network to simulate `fuji` locally. Therefore, the actually used sidecar info is in the local section...I assumed...

Reply:
Does it matter? Don't we just delete the config at the end of the test? Or are you wondering if we test the sidecar as well?